### PR TITLE
test: support for STREAMR_DOCKER_DEV_HOST env var

### PR DESCRIPTION
**Pull request by harbu**
Use same convention as in broker https://github.com/streamr-dev/broker/pull/233

### CODE REVIEW

File: test/integration/config.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ module.exports = {
         auth: {
             privateKey: process.env.ETHEREUM_PRIVATE_KEY || '0xe5af7834455b7239881b85be89d905d6881dcb4751063897f12be1b0dd546bdb',
         },
-        url: process.env.WEBSOCKET_URL || 'ws://localhost/api/v1/ws',
-        restUrl: process.env.REST_URL || 'http://localhost/api/v1',
+        url: process.env.WEBSOCKET_URL || `ws://${process.env.STREAMR_DOCKER_DEV_HOST || localhost}/api/v1/ws`,
+        restUrl: process.env.REST_URL || `http://${process.env.STREAMR_DOCKER_DEV_HOST || localhost}/api/v1`,
```
Comment (timoxley) [inline test/integration/config.js:12]: `localhost` probably should be quoted

File: test/integration/config.js [hunk 2]
```diff
@@ -22,14 +22,14 @@ module.exports = {
         storageNode: {
             // "broker-node-storage-1" on Docker environment
             address: '0xde1112f631486CfC759A50196853011528bC5FA0',
-            url: 'http://10.200.10.1:8891'
+            url: `http://${process.env.STREAMR_DOCKER_DEV_HOST || 10.200.10.1}:8891`
```
Comment (timoxley) [inline test/integration/config.js:25]: '10.200.10.1' should probably quoted

File: test/integration/config.js [hunk 3]
```diff
@@ -22,14 +22,14 @@ module.exports = {
...
         },
         sidechain: {
-            url: process.env.SIDECHAIN_URL || 'http://10.200.10.1:8546',
+            url: process.env.SIDECHAIN_URL || `http://${process.env.STREAMR_DOCKER_DEV_HOST || 10.200.10.1}:8546`,
```
Comment (timoxley) [inline test/integration/config.js:28]: Same

File: test/integration/config.js [hunk 4]
```diff
@@ -22,14 +22,14 @@ module.exports = {
...
             timeout: toNumber(process.env.TEST_TIMEOUT),
         },
         mainnet: {
-            url: process.env.ETHEREUM_SERVER_URL || 'http://10.200.10.1:8545',
+            url: process.env.ETHEREUM_SERVER_URL || `http://${process.env.STREAMR_DOCKER_DEV_HOST || 10.200.10.1}:8545`,
```
Comment (timoxley) [inline test/integration/config.js:32]: whoops

**Review by timoxley — CHANGES_REQUESTED** [review]
oh dear

File: test/integration/config.js (hunk 1 above)
Comment (harbu) [reply test/integration/config.js:12]: ah crap should've tested this before PR

**Comment by harbu** [comment]
Fixed quotes @timoxley

**Pull request merged**